app/views/site/why_indy.html.haml

```diff
@@ -3,44 +3,44 @@
+      = link_to 'Rob Annis (indywriter)', '//www.flickr.com/photos/indywriter/2379153276/'
       via Flickr
   :markdown
     So many reasons! We have big-time concert venues, sports like the Big Ten football and basketball championships, the NCAA tournament, conventions like GenCon, racing like the Indy 500 and the Brickyard 400, the Super Bowl. Yes, THE Super Bowl. We are an event city. We are Naptown no longer. We're also all of these:
 
-    * The **tech community** is strong, closely knit, and growing on a daily basis. It's accessible. It's exciting! (See [the IndyHackers Calendar](http://indyhackers.org/calendar))
-    * Developers are in **high demand** here. It's not uncommon for developers to get poached or for a successful company to swoop in and scoop up all the talent from a failed startup.(See [Indianapolis Programming Jobs](http://indyhackers.org/jobs) right here)
-    * We have a **variety** of local tech startups. People working on [online marketplaces](http://www.giggil.com/), people building web-based business apps ([there](http://instrumentalapp.com) [are](http://proposaljet.com) [a lot](http://docraptor.com/) [of these](http://tinderbox.com/)), people working in [robotics](http://blogs.wsj.com/venturecapital/2010/08/18/a-robot-lawn-mower-that-can-detect-darting-gophers/), people working in startup services, consultants who work on just about anything, college students and telecommuters for out of state biz who are here because of family, or because it’s cheap.
-    * The **cost of living** here is really low. You can get an apartment near Broad Ripple, arguably the heart of Indy's young professional hangout scene, for less than $700. With roommates it's even cheaper. If you're an entrepreneur, but not a mathematician: lower burn rate = longer runway. (See [Wikipedia](http://en.wikipedia.org/wiki/Indianapolis#Business_Climate_and_Real_Estate))
+    * The **tech community** is strong, closely knit, and growing on a daily basis. It's accessible. It's exciting! (See [the IndyHackers Calendar](//www.indyhackers.org/calendar))
+    * Developers are in **high demand** here. It's not uncommon for developers to get poached or for a successful company to swoop in and scoop up all the talent from a failed startup.(See [Indianapolis Programming Jobs](https://www.indyhackers.org/jobs) right here)
+    * We have a **variety** of local tech startups. People working on [online marketplaces](http://giggil.com/), people building web-based business apps ([there](https://instrumentalapp.com) are(http://proposaljet.com) [a lot](https://docraptor.com/) [of these](http://tinderbox.com/)), people working in [robotics](https://blogs.wsj.com/venturecapital/2010/08/18/a-robot-lawn-mower-that-can-detect-darting-gophers/), people working in startup services, consultants who work on just about anything, college students and telecommuters for out of state biz who are here because of family, or because it’s cheap.
```

Comment:
We need to update a few links on this line. These are all out of date:

http://www.giggil.com/
http://proposaljet.com
http://tinderbox.com/

Potentially we could replace them with more recent exciting startups